onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -867,15 +866,19 @@ PYBIND11_MODULE(onnxruntime_pybind11_state, m) {
 
 #endif
 
+  static std::unique_ptr<Environment> env;
   auto initialize = [&]() {
     // Initialization of the module
     ([]() -> void {
       // import_array1() forces a void return value.
       import_array1();
     })();
 
-    static std::unique_ptr<Environment> env;
-    OrtPybindThrowIfError(Environment::Create(env));
+    OrtPybindThrowIfError(Environment::Create(onnxruntime::make_unique<LoggingManager>(
+                                                  std::unique_ptr<ISink>{new CErrSink{}},
+                                                  Severity::kWARNING, false, LoggingManager::InstanceType::Default,
```

Comment:
This should probably be CLogSink for consistency with OrtEnv::GetInstance

Reply:
sure